todo/src/actions/index.js

```diff
@@ -0,0 +1,31 @@
+export const ADD = "ADD";
+export const TOGGLE = "TOGGLE";
+export const DELETETASK = "DELETETASK";
+export const DELETEALL = "DELETEALL";
```

Comment:
Typically, you see these 'constants' of sorts having any multi-word variables's words separated by underscores, like: `DELETE_ALL: 'DELETE_ALL'`. Having them not be separated like that is fine, I just want to make sure you know of this!